webui/src/pages/repositories/wizard/spark_config_step.jsx

```diff
@@ -24,14 +24,27 @@ async function uploadAndCommitReadme(repoId, branchName, importLocation) {
     await commits.commit(repoId, branchName, 'Add Spark quickstart README', {user: 'Spark quickstart'},);
 }
 
+async function uploadAndCommitSampleNotebook(repoId, branchName, importLocation) {
+    const SPARK_SAMPLE_NOTEBOOK_TEMPLATE_NAME = 'python.notebook.ipynb.tt';
+    const notebookProp = { repo: repoId, branch: branchName, ...lakeFSURLProp };
+    if (importLocation) {
+        notebookProp['import_location'] = importLocation;
+    }
+    const notebookConfig = await templates.expandTemplate(SPARK_SAMPLE_NOTEBOOK_TEMPLATE_NAME, notebookProp);
+    const notebookFile = new File([notebookConfig], 'Spark Demo.ipynb', {type: 'application/x-ipynb+json'});
+    await objects.upload(repoId, branchName, 'Spark Demo.ipynb', notebookFile);
+    await commits.commit(repoId, branchName, 'Added Spark Demo Notebook', {user: 'Spark quickstart'},);
```

Comment:
Getting a "no changes" error message when running the wizard, since this commit also catches the README.md file, which we try to commit later.
![image](https://user-images.githubusercontent.com/3149885/191701746-abcbe482-5c3d-4899-b786-bdc594eb9325.png)

Reply:
Good catch!
Fixed to have a single commit for both uploads.